### DISCUSSION TITLE
/Chatters will actually tell you how many people are currently chatting

### DISCUSSION THREAD

**Discussion by floodybYte** [Chatterino/chatterino2#3339] [Ideas]
Currently, as far as I know, if you type /Chatters in chat chatterino tells you how many people are logged in and watching the stream, aka are in the viewer list. What if /Chatters instead or additionally could check how many different people have typed in a timespan. This would make it much more useful.

**Comment by leon-richardt** [comment]
Sounds like an interesting idea, although it should definitely be in addition to the current behavior rather than replacing it.
